corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -82,11 +82,11 @@
           </li>
           <% } %>
           <% for (i = 0; i < pageCount; i++) { %>
-          <li class="page-item<% if (i === currentPage) { %> active<% } %>"><a class="page-link" data-id="<%= i %>"><%= i + 1 %></a></li>
+          <li class="page-item<% if (i === currentPage) { %> active<% } %>"><a class="page-link" aria-label="<%= `Page ${i + 1}` %>" <% if (i === currentPage ) { %> aria-current='page'<% } %> tabindex="0" data-id="<%= i %>"><%= i + 1 %></a></li>
```

Comment:
Ideally the aria-label would be translated. Usually we use underscore templating for js translations that involve string concatenation, something like `_.template(gettext("Page <%= page %>"))({page: i + 1})`

Reply:
OK. I ended up using a template function `pageNumLabel `as a template argument here because a direct approach with nested `<%= %>`'s was failing with a syntax error.